websites/docs-smoke-test/gatsby-config.js

```diff
@@ -8,11 +8,8 @@ const {
 } = require('@commercetools-docs/gatsby-theme-docs/configure-theme');
 
 module.exports = {
-  // https://www.gatsbyjs.com/docs/reference/release-notes/v2.28/#feature-flags-in-gatsby-configjs
-  // https://www.gatsbyjs.com/docs/reference/release-notes/v2.30
   flags: {
-    FAST_DEV: true,
-    PRESERVE_FILE_DOWNLOAD_CACHE: true,
+    DEV_SSR: true,
```

Comment:
More explicit than FAST_DEV (which implies DEV_SSR).  esp. over version changes we don't want implicit flag changes